test/support/helpers/review-result.helper.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+
+/**
+ * @module ReviewResultHelper
+ */
+
+const { generateUUID } = require('../../../app/lib/general.lib.js')
+const ReviewResultModel = require('../../../app/models/review-result.model.js')
+
+/**
+ * Add a new review result record for 2pt matching
+ *
+ * If no `data` is provided, default values will be used. These are
+ *
+ * - `id` - [random UUID]
+ * - `billRunId` - [random UUID]
+ * - `licenceId` - [random UUID]
+ * - `chargeVersionId` - [random UUID]
+ * - `chargeReferenceId` - [random UUID]
+ * - chargePeriodStartDate - 2022-04-01
+ * - chargePeriodEndDate - 2022-06-05
+ * - chargeVersionChangeReason - 'Strategic review of charges (SRoC)'
+ * - `reviewChargeElementResultId` - [random UUID]
+ * - `reviewReturnResultId` - [random UUID]
+ * - updatedAt - new Date()
```

Comment:
The `created_at` and `updated_at` get automatically populated when the record is created. So, we shouldn't need to default them.